README.adoc

```diff
@@ -105,6 +105,19 @@ $ oadm policy add-role-to-user edit \
         -n openshift-infra
 ----
 
+=== Hawkular Service Account
+
+In order for Hawkular Metrics to cluster, it need to be able to gather information about the other pods running in its service. In order to do so it requires the `view` permission.
+
+To add the `view` permission to the `hawkular` servivce account run the following command:
```

Comment:
Run the following command to add the `view` permission to the `hawkular` service account: